argh/tests/lib.rs

```diff
@@ -1496,18 +1354,30 @@ fn subcommand_does_not_panic() {
         argh::EarlyExit { output: "no subcommand name".into(), status: Err(()) },
     );
 
+    assert_eq!(
+        SubCommandEnum::help_json_from_args(&[]).unwrap_err(),
```

Comment:
I see 3 tests directly calling `help_json_from_args()` and expecting an `EarlyExit` error. Can you add at least one more test showing what a successful call to `help_json_from_args()` looks like?

Reply:
I added many tests in argh/tests/help_json.rs which test the various positive use cases. These are to make sure the help_json_from_args calls do not panic as well.